### ISSUE TITLE
Detecting user's browser language

### ISSUE THREAD

**Issue by highersource** [baptisteArno/typebot.io#463]
Hi, I'm trying to set up a way of detecting the user's browser language on my bot. I'm using JavaScript `(new Intl.NumberFormat()).resolvedOptions().locale` but have also tried `navigator.language` with the same results.

I've used the "Script" block to create a simple function to detect the user's browser language:

```
var Language = (new Intl.NumberFormat()).resolvedOptions().locale

if (Language.includes('en')) {
  alert('user language is English')
} else {
  alert('user language is not English')
}
```

This works and I get a different alert message on my browser, depending on which language is set to.

Now I want to use this same logic through Typebot's "Set Variable" and "Condition" blocks like this:

_Set Variable block:_
`Language = (new Intl.NumberFormat()).resolvedOptions().locale`

_Condition block:_
`Language contains "en"` → outcome 1
`else` → outcome 2

But this doesn't work, I'm always getting "outcome 1" even if I set my browser and system language to anything other than English. 

Why is the pure JS alert message working fine but not the Typebot integration? Is there another way of doing this? Thanks!

**Comment by baptisteArno** [comment]
That's because the Set variable code is executed on the server by default now. I recently added an option to execute it on the front-end (#461). You'll just have to check the option on the Set variable block settings.

I just released v2.12.1. Can you pull it and tell me if that works fine for you?

**Comment by satoshinotdead** [comment]
Sorry but can I see an complete implementation? I'm looking for adding multi-language support to Typebot 🧡
